### PR TITLE
Added .aicoe-ci.yaml file

### PR DESCRIPTION
**Pull request by suppathak**
After creating a repository on quay.io (https://github.com/operate-first/continuous-delivery/issues/32#issuecomment-1076472447). I have then updated the config file.

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+  - thoth-build
+
+build:
+  build-stratergy: Source
+  build-source-script: "image:///opt/app-root/builder"
+  base-image: quay.io/thoth-station/s2i-custom-notebook:latest
```
Comment (harshad16) [inline .aicoe-ci.yaml]: is this application python 3.6 based ?
if not please use s2i-custom-py38-notebook

**Review by harshad16 — CHANGES_REQUESTED** [review]
one change required

File: .aicoe-ci.yaml (hunk 1 above)
Comment (suppathak) [reply .aicoe-ci.yaml]: made the changes, thanks :+1:

**Pull request merged**